visualdl/io/bfile.py

```diff
@@ -31,6 +31,11 @@ def register_filesystem(self, path, filesystem):
         self._register_factories.update({path: filesystem})
 
     def get_filesystem(self, path):
+        if 'hdfs://' in path and "hdfs" not in self._register_factories.keys():
```

Comment:
`path.startswith('hdfs://')` makes much sense?

Reply:
Professional advice, i will modify it.